app/views/declaration/destinationCountries/routing_countries_summary.scala.html

```diff
@@ -0,0 +1,74 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import controllers.declaration.routes
+@import controllers.navigation.Navigator
+@import forms.declaration.RoutingQuestion
+@import forms.declaration.RoutingQuestion.{no, RoutingQuestionPage, yes}
+@import models.requests.JourneyRequest
+@import views.components.inputs.RadioOption
+@import views.Title
+
+@this(main_template: views.html.main_template)
+
+@(mode: Mode, form: Form[RoutingQuestion], countriesOfRouting: Seq[String])(implicit request: JourneyRequest[AnyContent], messages: Messages)
+
+@main_template(
+    title = Title("declaration.routingCountries.summary.title", "declaration.routingCountries.summary.heading")
+) {
+
+    @components.back_link(Navigator.backLink(RoutingQuestionPage)(request)(mode), mode)
+
+    @components.error_summary(form.errors)
+
+    @components.heading(
+        title = messages("declaration.routingCountries.summary.header", countriesOfRouting.size),
+        sectionHeader = messages("declaration.routingCountries.summary.heading")
+    )
+
+    <table id="routing-countries-table" class="form-group">
+        <tbody>
```

Comment:
missing `thead` ?

Reply:
There is no `thead`. Not sure what should I do in this case

Reply:
Ok, design looks like this is table without header row